AI-and-Analytics/Features-and-Functionality/IntelPython_GPU_numba-dpex_Genetic_Algorithm/README.md

```diff
@@ -22,103 +22,133 @@ In this sample, you will create and run the general genetic algorithm and optimi
-
-#### Activate Conda
-
-1. Activate the Conda environment.
-   ```
-   conda activate base
-   ```
-   By default, the AI Kit is installed in the `/opt/intel/oneapi` folder and requires root privileges to manage it.
-
-   You can choose to activate Conda environment without root access. To bypass root access to manage your Conda environment, clone and activate your desired Conda environment using the following commands similar to the following.
-
-   ```
-   conda create --name usr_base --clone base
-   conda activate usr_base
-   ```
-
-#### Run the Python Script
-
-1. Change to the sample directory.
-2. Run the script.
-   ```
-   python IntelPython_GPU_numba-dpex_Genetic_Algorithm.py
-   ```
-
-#### Run the Jupyter Notebook (Optional)
-
-1. Launch Jupyter Notebook.
-   ```
-   jupyter notebook --ip=0.0.0.0
-   ```
-2. Follow the instructions to open the URL with the token in your browser.
-3. Locate and select the Notebook.
-   ```
-   IntelPython_GPU_numba-dpex_Genetic_Algorithm.ipynb
-   ```
-4. Run every cell in the Notebook in sequence.
-
-#### Troubleshooting
-
-If you receive an error message, troubleshoot the problem using the **Diagnostics Utility for Intel® oneAPI Toolkits**. The diagnostic utility provides configuration and system checks to help find missing dependencies, permissions errors, and other issues. See the [Diagnostics Utility for Intel® oneAPI Toolkits User Guide](https://www.intel.com/content/www/us/en/develop/documentation/diagnostic-utility-user-guide/top.html) for more information on using the utility.
+This sample code is implemented for GPUs using Python. The sample assumes you have numba-dpex installed inside a Conda environment, similar to what is installed with the Intel® Distribution for Python*.
+
+>**Note**: Read *[Get Started with the Intel® Distribution for Python*](https://www.intel.com/content/www/us/en/developer/articles/technical/get-started-with-intel-distribution-for-python.html)* to find out how you can achieve performance gains through Intel optimizations.
+
+The sample tutorial contains one Jupyter Notebook and one Python script. You can use either.
+
+## Environment Setup
+You will need to download and install the following toolkits to use the sample.
+<!-- Use numbered steps instead of subheadings -->
+
+**1. Get Intel® Distribution for Python***
+
+If you have not already, install Intel® Distribution for Python* via [Installer](https://www.intel.com/content/www/us/en/developer/tools/oneapi/distribution-python-download.html?operatingsystem=linux&linux-install-type=offline). AI and Analytics samples are validated on AI Tools Offline Installer. It is recommended to select Offline Installer option in AI Tools Selector.
+
+**2. (Offline Installer) Activate the AI Tools bundle base environment**
```

Comment:
For IDP the only option is offline installer, so no need to additionally specify this in brackets
```suggestion
**2. Activate the AI Tools bundle base environment**
```